test/removeNPMAbsolutePaths.js

```diff
@@ -406,5 +403,34 @@ describe('removeNPMAbsolutePaths.js', function () {
           expect(writeFile).to.have.been.calledOnce.and.calledWith(filePath);
         });
     });
+
+    it('checks that value of file contents do not contain specific removed fields', function () {
```

Comment:
The name should say what you are validating not how you validate it.

This test should go into 
```
describe('removeNPMAbsolutePaths.js', function () {
  ...
  describe('is valid file path', function () {
    ....
    it('removes only the specified fields if the --fields option is used', function () {
      <TEST_HERE>
```